caffeinated/admin/extend/registrations/Extend_Registrations_Admin_Page.core.php

```diff
@@ -1156,16 +1156,16 @@ protected function _event_registrations_list_table()
         );
         $this->_template_args['after_list_table'] = $this->_display_legend($legend_items);
         $event_id = isset($this->_req_data['event_id']) ? $this->_req_data['event_id'] : null;
-        $this->_template_args['before_list_table'] = ! empty($event_id)
+        /** @var EE_Event $event */
+        $event = EEM_Event::instance()->get_one_by_ID($event_id);
+        $this->_template_args['before_list_table'] = ! empty($event_id) && $event instanceof EE_Event
```

Comment:
You can simplify this here:

```suggestion
        $this->_template_args['before_list_table'] = $event instanceof EE_Event
```